pkg/apis/flow/v1alpha1/jqtransformation_types.go

```diff
@@ -53,15 +53,9 @@ type JQTransformationSpec struct {
 	duckv1.SourceSpec `json:",inline"`
 }
 
-// JQTransformationStatus communicates the observed state of the JQTransformation (from the controller).
-type JQTransformationStatus struct {
-	duckv1.SourceStatus  `json:",inline"`
-	duckv1.AddressStatus `json:",inline"`
-}
```

Comment:
This type wasn't used anywhere, so I used the occasion to delete it.

Reply:
ok